packages/did-core-test-server/suites/did-spec/default.json

```diff
@@ -2,6 +2,49 @@
   {
     "name": "did-spec",
     "didMethods": {
+      "did:key - 2018 Ed25519 cryptosuite - did-key-js - Digital Bazaar": {
```

Comment:
I expect this file to get very unwieldy -- we have multiple implementations coming down the pipeline -- can we break it up into implementations?

Reply:
yeah, we may want to switch to `.js` and use require to build the json instead... like we have done here:

https://github.com/w3c-ccg/vc-http-api/blob/main/packages/vc-http-api-test-server/config/Transmute.js#L43

Reply:
> yeah, we may want to switch to `.js` and use require to build the json instead...

+1, let's do that before we get too many new entries and it becomes major surgery.